src/scripts/components/globe-navigation/globe-navigation.tsx

```diff
@@ -32,11 +34,12 @@ const GlobeNavigation: FunctionComponent = () => {
   return (
     <div className={styles.globeNavigation}>
       <Button label={label} onClick={() => onProjectionHandler()} />
-      <Button
-        className={styles.compassIcon}
-        icon={CompassIcon}
+      <div
+        className={styles.compass}
         onClick={() => dispatch(setFlyToAction({...defaultView}))}
-      />
+        style={{transform: `rotate(${globecheck.position.longitude}deg)`}}>
```

Comment:
Using the longitude seems wrong because the north arrow of the icon rotates even if the globe's north is up. One of the camera rotation values could work better